lib/readyset/explain.rb

```diff
@@ -0,0 +1,59 @@
+module Readyset
+  class Explain
+    attr_reader :id, :text, :supported
+
+    # Gets information about the given query from ReadySet, including whether it's supported to be
+    # cached, its current status, the rewritten query text, and the query ID.
+    #
+    # The information about the given query is retrieved by invoking `EXPLAIN CREATE CACHE FROM` on
+    # ReadySet.
```

Comment:
_**typo:**_ Module/Class responsibility descriptions should go above `class Explain`.